scripts/dev-env.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -euo pipefail
+target="$1"
+
+if ! command -v bw > /dev/null; then
+    echo "You need the bitwarden cli tool installed:"
+    echo 
+    echo "    https://bitwarden.com/help/article/cli/#download-and-install"
+    echo
+    exit 1
+fi
+
+if test -z "$BW_SESSION"; then
+    echo "Unlocking bitwarden..."
+    BW_SESSION=$(bw unlock --raw)
+    export BW_SESSION
+fi
+
+SOCIAL_AUTH_BW_ID=d48e3e3f-07ed-4865-b348-ace700ee004
+GH_DEV_TOKEN_ID=10242708-234a-4162-982f-ace700ee5c53
```

Comment:
Small nit: one contains `BW`, one doesn't.  I'm not sure we need the `BW`, but up to you 🙂

Reply:
updated